packages/mobile/src/components/Avatar.tsx

```diff
@@ -49,20 +45,51 @@ export function getE164Number(e164Number?: string, recipient?: Recipient) {
 }
 
 export function Avatar(props: Props) {
```

Comment:
Only tangentially related but can we extend `Avatar`'s use to the `DrawerNavigator` as well? Tbh that is where I was expecting to see this component but was surprised to find that we are essentially recreating it there

Reply:
It's similar, just style differences I believe but I made a change on this PR: https://github.com/celo-org/celo-monorepo/pull/6522 which adds an extra button next to the image and would make it incompatible with using the `Avatar`. Not sure how to go around that.

Reply:
Cool I was just thinking out loud...sounds like it makes sense to leave it as is